tests/common/fixtures/duthost_utils.py

```diff
@@ -624,6 +624,10 @@ def wait_bgp_sessions(duthost, timeout=120):
     A helper function to wait bgp sessions on DUT
     """
     bgp_neighbors = duthost.get_bgp_neighbors_per_asic(state="all")
+    if duthost.is_supervisor_node():
```

Comment:
on voq chassis supervisor has no bgp sessions, can we modify the caller to send the needed timeout when ` wait_bgp_sessions` is called for Linecards